packages/ui/report.api.md

```diff
@@ -591,6 +591,8 @@ export const componentDefinitions: {
     readonly classNames: {
       readonly tooltip: 'bui-Tooltip';
       readonly arrow: 'bui-TooltipArrow';
+      readonly arrowFill: 'bui-TooltipArrowFill';
```

Comment:
Worth discussing a bit whether these classes should be hidden from the public API.